src/NServiceBus.Metrics.PerformanceCounters/PerformanceCounterUpdater.cs

```diff
@@ -46,5 +66,34 @@ public void Observe(ProbeContext context)
 
     readonly PerformanceCountersCache cache;
     readonly Dictionary<string, CounterInstanceName?> legacyInstanceNameMap;
+    readonly ConcurrentDictionary<CounterInstanceName, IPerformanceCounterInstance> expirable = new ConcurrentDictionary<CounterInstanceName, IPerformanceCounterInstance>();
```

Comment:
How about we change the name to `resettable`? This would align with `resetEvery` vairable name